docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -716,6 +777,222 @@ Alter TABLE PRODUCT
 ALTER COLUMN vend_id RENAME TO cstm_id;
 ```
 
+* The following table _orders_ has two normal indexes 

```

Comment:
Suggest "non-unique" instead of "normal"

Reply:
Thanks @DaveBirdsall, I've udpated :)